app/i18n/en/conf.php

```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',
-		'theme' => 'Theme',
+		'theme' => array(
+			'_' => 'Theme',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
+				'description' => 'This theme is deprecated and will be not available anymore in a <a href=https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme target=_blank>future release of FreshRSS</a>',	// TODO
```

Comment:
Deprecated is programmer jargon, better to paraphrase.
```suggestion
				'description' => 'This theme is no longer supported and will be not available anymore in a <a href=https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme target=_blank>future release of FreshRSS</a>',	// TODO
```

Reply:
Should we rephrase this too? 
![grafik](https://user-images.githubusercontent.com/1645099/211660432-87f6442f-ff9a-4f35-9d87-914611685d0f.png)

Reply:
The same would apply, but "deprecated and will be removed" is much clearer than "deprecated and will be not available anymore".

So `no longer supported and will be removed` is better for both.